doc/classes/@GlobalScope.xml

```diff
@@ -2141,135 +2141,36 @@
 		<constant name="JOY_BUTTON_DPAD_RIGHT" value="14" enum="JoyButtonList">
 			Game controller SDL D-pad right button.
 		</constant>
-		<constant name="JOY_SDL_BUTTONS" value="15" enum="JoyButtonList">
+		<constant name="JOY_BUTTON_SDL_MAX" value="15" enum="JoyButtonList">
 			The number of SDL game controller buttons.
 		</constant>
-		<constant name="JOY_SONY_X" value="0" enum="JoyButtonList">
-			Sony DualShock controller X button maps to SDL button A.
-		</constant>
```

Comment:
That's busy work but I'd edit the descriptions of the SDL mappings to also include a reference to the Sony, Xbox and Nintendo equivalent. E.g.:
```
<constant name="JOY_BUTTON_A" value="0" enum="JoyButtonList">
	Game controller SDL button A.
</constant>
```
becomes:
```
<constant name="JOY_BUTTON_A" value="0" enum="JoyButtonList">
	Button A on abstracted SDL game controller.
	Also corresponds to: Button A on Xbox controller, Cross button on Sony DualShock controller, Button B on Nintendo controller.
</constant>
```
And so on.

That's busy work so since you've done a lot already, I can do it in a follow PR if you prefer.